src/pages/plans/Plans.css

```diff
@@ -0,0 +1,79 @@
+.plansContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
Esto lo pondria como `row` ya que al tenerlo en `column` esta haciendo que esten desalineados horizontalmente el selector del formulario

<img width="1023" height="433" alt="Image" src="https://github.com/user-attachments/assets/45734a82-bac9-4fc8-aba7-41df4298cd3e" />



```suggestion
  flex-direction: row;
```

Reply:
Ademas, agregaria una media-query para que en modo responsive (para un celular) la direccion si sea column para que aparezca el selector arriba y el formulario abajo

Reply:
Okey, lo había hecho así a propósito porque en la foto del diseño está un poco más arriba el select
<img width="1207" height="605" alt="image" src="https://github.com/user-attachments/assets/ab4f6d85-3a99-4267-9354-31d6208f9446" />

Reply:
Sip, es cierto, pero queda medio feo así que mejor con el cambio 

<img width="194" height="259" alt="image" src="https://github.com/user-attachments/assets/9e09c9d2-a407-48e0-8795-8af25b306c32" />